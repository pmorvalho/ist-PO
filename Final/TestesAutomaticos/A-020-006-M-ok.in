8
4
1
9
0
P1
0
0
7
P1
4
8
4
1
11
0
0
0
4
7
P1
0
