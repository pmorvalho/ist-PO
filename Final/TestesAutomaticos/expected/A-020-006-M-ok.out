Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Identificador da secção: Nova secção activa: '1'.
Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Número do parágrafo: Identificador único: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Identificador do elemento de texto: Do or do not.
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: Objects in Space
Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
Secções de topo: 3
Dimensão do documento (bytes): 269
Identificadores únicos: 3
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Identificador da secção: Nova secção activa: '1'.
Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Número do parágrafo: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: Objects in Space
Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
Secções de topo: 3
Dimensão do documento (bytes): 256
Identificadores únicos: 2
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Identificador do elemento de texto: O elemento de texto 'P1' não existe.
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: 